06-building-a-program-start-to-finish/solving-the-problem/output/inverted-index.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+
```

Comment:
Also creating a class without exporting flags a violation. So i ignored that too